infinit/content.md

```diff
@@ -0,0 +1,67 @@
+Infinit’s storage platform transforms commodity servers into multiple flexible, scalable, secure and fault-tolerant storage logics tailored to your applications.
```

Comment:
You probably want to include an `h1` header above this, something like `# Infinit` or `# What is Infinit?` (similar to other images) or else this block will combine with the block in the template above it.  If you run `./update.sh infinit` it should generate `infinit/README.md` which you can then use to get an idea for how your generated content is going to flow (just don't commit that generated file -- we've got a bot that'll update it automatically after merge).

Also, that `logo.png` that's included here will only show up if `%%LOGO%%` is used somewhere here in the body.  In other descriptions, we usually put it right after this initial "what is XYZ?" paragraph, but some maintainers prefer to list it either earlier or later -- it's really up to you. :+1: